airflow/providers/apache/beam/provider.yaml

```
@@ -42,7 +42,7 @@ versions:
 
 dependencies:
   - apache-airflow>=2.3.0
-  - apache-beam>=2.33.0
+  - apache-beam==2.47.0rc1
```

```suggestion
  - apache-beam>=2.47.0rc1
```
@potiuk Should it be this instead?

---

Finally it should be likely >= 2.47 when they release. For now - I wanted to make sure it works, and put ==  to unblock @lwyszomi and the team to start solving test failures as the image could not be built without it due to conflicts.

And we are in a good place finally. The image builds without conflicts :tada: !!! and those are the upgraded packages  -- exactly what we wanted :) 

```
< # This constraints file was automatically generated on 2023-04-27T21:53:27Z
---
> # This constraints file was automatically generated on 2023-04-27T23:10:05Z
43a44
> Shapely==1.8.5.post1
63c64
< apache-beam==2.46.0
---
> apache-beam==2.47.0rc1
108c109
< black==23.1a1
---
> black==23.3.0
162c163
< elasticsearch-dbapi==0.2.10
---
> elasticsearch-dbapi==0.2.2
164c165
< elasticsearch==7.13.4
---
> elasticsearch==7.17.9
189,190c190,192
< google-api-core==2.8.2
< google-api-python-client==1.12.11
---
> google-ads==21.0.0
> google-api-core==2.11.0
> google-api-python-client==2.86.0
192c194
< google-auth-oauthlib==0.8.0
---
> google-auth-oauthlib==1.0.0
194,204c196,206
< google-cloud-aiplatform==1.16.1
< google-cloud-appengine-logging==1.1.3
< google-cloud-audit-log==0.2.4
< google-cloud-automl==2.8.0
< google-cloud-bigquery-datatransfer==3.7.0
< google-cloud-bigquery-storage==2.14.1
< google-cloud-bigquery==2.34.4
< google-cloud-bigtable==2.11.1
< google-cloud-build==3.9.0
< google-cloud-compute==0.7.0
< google-cloud-container==2.11.1
---
> google-cloud-aiplatform==1.24.1
> google-cloud-appengine-logging==1.3.0
> google-cloud-audit-log==0.2.5
> google-cloud-automl==2.11.1
> google-cloud-bigquery-datatransfer==3.11.1
> google-cloud-bigquery-storage==2.19.1
> google-cloud-bigquery==3.10.0
> google-cloud-bigtable==2.17.0
> google-cloud-build==3.16.0
> google-cloud-compute==1.11.0
> google-cloud-container==2.21.0
206,225c208,227
< google-cloud-datacatalog==3.9.0
< google-cloud-dataflow-client==0.5.4
< google-cloud-dataform==0.2.0
< google-cloud-dataplex==1.1.0
< google-cloud-dataproc-metastore==1.6.0
< google-cloud-dataproc==5.0.0
< google-cloud-dlp==3.8.0
< google-cloud-kms==2.12.0
< google-cloud-language==1.3.2
< google-cloud-logging==3.2.1
< google-cloud-memcache==1.4.1
< google-cloud-monitoring==2.11.0
< google-cloud-orchestration-airflow==1.4.1
< google-cloud-os-login==2.7.1
< google-cloud-pubsub==2.13.5
< google-cloud-redis==2.9.0
< google-cloud-resource-manager==1.6.0
< google-cloud-secret-manager==1.0.2
< google-cloud-spanner==1.19.3
< google-cloud-speech==1.3.4
---
> google-cloud-datacatalog==3.12.0
> google-cloud-dataflow-client==0.8.3
> google-cloud-dataform==0.5.1
> google-cloud-dataplex==1.4.3
> google-cloud-dataproc-metastore==1.11.0
> google-cloud-dataproc==5.4.1
> google-cloud-dlp==3.12.1
> google-cloud-kms==2.16.1
> google-cloud-language==2.9.1
> google-cloud-logging==3.5.0
> google-cloud-memcache==1.7.1
> google-cloud-monitoring==2.14.2
> google-cloud-orchestration-airflow==1.7.1
> google-cloud-os-login==2.9.1
> google-cloud-pubsub==2.16.0
> google-cloud-redis==2.12.1
> google-cloud-resource-manager==1.10.0
> google-cloud-secret-manager==2.16.1
> google-cloud-spanner==3.32.0
> google-cloud-speech==2.19.0
227,232c229,234
< google-cloud-tasks==2.10.1
< google-cloud-texttospeech==1.0.3
< google-cloud-translate==1.7.2
< google-cloud-videointelligence==1.16.3
< google-cloud-vision==1.0.2
< google-cloud-workflows==1.7.1
---
> google-cloud-tasks==2.13.1
> google-cloud-texttospeech==2.14.1
> google-cloud-translate==3.11.1
> google-cloud-videointelligence==2.11.1
> google-cloud-vision==3.4.1
> google-cloud-workflows==1.10.1
235c237
< googleapis-common-protos==1.56.4
---
> googleapis-common-protos==1.59.0
239c241
< grpc-google-iam-v1==0.12.4
---
> grpc-google-iam-v1==0.12.6
241c243
< grpcio-status==1.48.2
---
> grpcio-status==1.54.0
356,357c358,359
< packaging==21.3
< pandas-gbq==0.17.9
---
> packaging==23.1
> pandas-gbq==0.19.1
385,386c387,388
< proto-plus==1.19.6
< protobuf==3.20.0
---
> proto-plus==1.22.2
> protobuf==4.22.3
394c396
< pyarrow==9.0.0
---
> pyarrow==11.0.0
512c514
< tableauserverclient==0.24
---
> tableauserverclient==0.25
558c560
< uritemplate==3.0.1
---
> uritemplate==4.1.1
```